src/main/resources/javaagent.conf

```diff
@@ -19,6 +19,7 @@ service {
 graphite {
   hostname = ${?GRAPHITE_HOST}
   port = ${?GRAPHITE_PORT}
+  protocol = ${?GRAPHITE_PROTOCOL}
```

Comment:
This pull request made me a realize we need to refactor a method we currently have in the JVM Agent flavor. I'll do it in a separate pull request. For now, just add your config there:
Jmx2GraphiteJavaAgent.getArgumentConfigurationRepresentation for the JVM agent to support this feature as well.